pkg/api/types.go

```diff
@@ -928,6 +932,16 @@ type VsphereVirtualDiskVolumeSource struct {
 	FSType string `json:"fsType,omitempty"`
 }
 
+// Represents a Photon Controller persistent disk resource.
+type PhotonPersistentDiskVolumeSource struct {
+	// ID that identifies Photon Controller persistent disk
+	PdID string `json:"pdID"`
```

Comment:
Same reply as the first one. Disk name is not unique so the user should give ID instead of name.